test/mechanics/timeIntegration/TimeControl.cpp

```diff
@@ -0,0 +1,92 @@
+#include "BoostUnitTest.h"
+
+#include "mechanics/timeIntegration/TimeControl.h"
+
+#include <iostream>
+
+using namespace NuTo;
+
+// %%% Helper functions %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+bool CheckExceptionMessageCorrect(MechanicsException const& ex,std::string expectedMsg)
```

Comment:
This is just a workaround. Just wait what we decide concerning the exception issue